packages/google_sign_in/google_sign_in/lib/google_sign_in.dart

```diff
@@ -259,26 +253,25 @@ class GoogleSignIn {
   ///
   /// At most one in flight call is allowed to prevent concurrent (out of order)
   /// updates to [currentUser] and [onCurrentUserChanged].
-  Future<GoogleSignInAccount> _addMethodCall(String method) async {
+  Future<GoogleSignInAccount> _addMethodCall(
```

Comment:
nits(optional): Add doc for the new parameter 'canSkipCall`.

Reply:
Added!